discovery-provider/src/tasks/entity_manager/entity_manager.py

```diff
@@ -130,6 +140,7 @@ def entity_manager_update(
                     logger.info(
                         f"entity_manager.py | failed to process tx error {e} | with event {event}"
                     )
+                    metric_num_errors.save(1, {"entity_type": params.entity_type})
```

Comment:
Let's have this metric be captured before the exception as well, but with a different scope.

So instead of having an "errors" metric, let's have a "processed transactions" metric where we have two scopes: "processed" and "failed".